src/odemis/acq/move.py

```diff
@@ -614,6 +618,437 @@ def _doCryoSwitchSamplePosition(self, future, target):
                 future._task_state = FINISHED
 
 
+class ConvertStage:
```

Comment:
As discussed, instead of adding the convert stage, move the transformation into the posture manger, like https://github.com/delmic/odemis/pull/2893